products/workers/src/content/platform/pricing.md

```diff
@@ -5,98 +5,152 @@ pcx-content-type: concept
+
+<TableWrap>
+|               | Paid plan                                                    |
+| ------------- | ------------------------------------------------------------ |
+| Requests      | 1 million, + $0.15/million                                   | 
+| Duration      | 400,000 GB-s, + $12.50/million GB-s<sup>1</sup><sup>2</sup>  |
+
+    
 </TableWrap>
+1. Duration is billed in wall-clock time as long as the Object is active, but is shared across all requests active on an Object at once.  Once your Object stops receiving requests, it will be removed from memory and stop incurring duration charges. A WebSocket being connected to the Durable Object counts as the Object being active.
+2. Duration billing charges for the 128 MB of memory your Durable Object is allocated, regardless of actual usage.  If your account creates many instances of a single Durable Object class, Durable Objects may run in same isolate on the same physical machine and share the 128MB of memory. These Durable Objects are still billed as if they are allocated a full 128MB of memory.
 
-1. Cloudflare will bill for Duration charges based on the higher of your wall time or CPU time, with a multiple applied to the CPU time to account for the processing power allotted to your script. We will not bill for wall time Duration charges beyond the execution [limit](/platform/limits#worker-limits) given.
-2. Unbound egress data transfer fees can be reduced when sending data to members of the [Bandwidth Alliance](https://www.cloudflare.com/bandwidth-alliance/).
+### Durable Objects billing examples
 
-### Worker billing examples
+These examples exclude the costs for the Workers calling the Durable Objects.
 
 #### Example 1
 
-If an Unbound Worker executed 1.5 million times, used a total of 200,000 GB-s, and sent a total egress data of approximately 6 GB, the estimated cost in a month would be:
+If a single Durable Object was called by a Worker 1.5 million times, and was active for 1,000,000 seconds in the month, the estimated cost in a month would be:
 
-Total = ~$0.12 USD + Minimum $5/mo usage = **$5.12**
+Total = ~$0.8 USD + Minimum $5/mo usage = **$5.08**
 
 - (1.5 million requests - included 1 million requests) x $0.15 / 1,000,000 = **$0.075**
-- (200,000 GB-s - included 400,000 GB-s) x $12.50 / 1,000,000 = **$0.00**
-- (6 GB - included 5 GB) x 0.045 = **$0.045**
+- 1,000,000 seconds * 128 MB / 1 GB = 128,000 GB-s
+- (128,000 GB-s - included 400,000 GB-s) x $12.50 / 1,000,000 = **$0.00**
 
 #### Example 2
 
-If an Unbound Worker executed 5 million times, used a total of 800,000 GB-s, and sent a total egress data of approximately 20 GB, the estimated cost in a month would be:
+If 100 Durable Objects each had 100 WebSocket connections established to each of them which sent approximately 1 message a minute for a month, the estimated cost in a month would be, if the messages overlapped so that the Objects were actually active for half the month:
```

Comment:
```suggestion
If 100 Durable Objects each had 100 WebSocket connections established to each of them which sent approximately one message a minute for a month, the estimated cost in a month would be, if the messages overlapped so that the Objects were actually active for half the month:
```

Is there supposed to be something after "in a would would be ____"?